__tests__/app.test.js

```diff
@@ -25,4 +26,30 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJSON);
       });
   });
+
+  test("404: not found", () => {
+    return request(app)
+      .get("/notURL")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+});
+
+describe("GET /api/topics", () => {
+  test("200 : responds with all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toHaveLength(data.topicData.length);
```

Comment:
I like that you've tested the length like this, it may be worth me pointing out that you can also just use a number here because you know how many topics there should be from looking at the data. However, this way can be handy if you were to change the data later for whatever reason.